src/renderers/dom/shared/ReactDOMComponent.js

```diff
@@ -200,6 +201,10 @@ var newlineEatingTags = {
   'textarea': true
 };
 
+var wrappers = {
+  'input': ReactDOMInput
```

Comment:
Yea, I wanted to fix all of these callsites. We have a bunch all over the code.

It should be `new Set([ key, key, key... ])` or `new Map([[ key, value ], ...])` but since we can't rely on ES6 Maps/polyfills just yet, we can make our own helpers. Like `ConstantSet` and `ConstantMap` that just creates a frozen keyed object for example. (Freezing in prod might actually be beneficial for this case since it is not called in a hot path.)

An alternative pattern is ofc a function with a big switch statement which I would prefer for code paths. I don't care as much for raw data if it is things like DOM strings that have to be around at runtime anyway.

However, if it is internal enums etc. we should probably not use maps/sets to manage them.